source/renderer/app/config/stakingStakePools.dummy.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "controlledStake": 7,
+    "controlledStake": 2938470000000,
```

Comment:
@daniloprates this is the `controlledStake` shown in Storybook - can you please update `generatStakePoolsFakeData` and generate a new dummy file with higher range of values for `controlledStake` and for `producedBlocks`? We need to test our short numer formatting!

Reply:
@yakovkaravelov will take care of this. Thanks!